code/datums/elements/mirage_border.dm

```diff
@@ -0,0 +1,42 @@
+/**
+ * Creates a mirage effect allowing you to see around the world border, by adding the opposite side to its vis_contents.
+ */
+/datum/element/mirage_border
+
+/datum/element/mirage_border/Attach(datum/target, turf/target_turf, direction, range=world.view)
+	. = ..()
+	if(!isturf(target))
+		return COMPONENT_INCOMPATIBLE
+	if(!target_turf || !istype(target_turf) || !direction)
+		. = COMPONENT_INCOMPATIBLE
```

Comment:
```suggestion
		return ELEMENT_INCOMPATIBLE
	if(!target_turf || !istype(target_turf) || !direction)
		. = ELEMENT_INCOMPATIBLE
```
The values are the same, but because they're two distinct datums, the defines differ.

Reply:
Oops. I changed these but forgot to port it.